src/components/adaptive-cards/Component/Component.scss

```diff
@@ -1,3 +1,46 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-component;
+
+.#{$C}__container--horizontal {
+  display: flex;
+
+  > .#{$C}__separator--true:not(:first-child) {
```

Comment:
if the user wants a separator on the first child, who are we to deny that

Reply:
I see that the adaptivecards.io/designer ignores spacing for the first child, so please ignore my prev comment